### DISCUSSION TITLE
Campo email es requerido?

### DISCUSSION THREAD

**Discussion by frithmatias** [mercadopago/sdk-js#121] [Q&A] [answered]
Estoy implementando checkout Bricks y quiero sacar el campo email ya que muchos pagadores con tarjeta no tienen cuenta de mercadopago, ¿es posible quitar este campo en el cardform?

**Comment by hdlopez** [comment, marked as answer]
Hola Matías! Para pagar mediante un brick el usuario no necesita cuenta de mercadopago (simplemente necesita especificar su correo para recibir la confirmación o estado de la compra).

Es importante además mencionar que el campo mail es requerido, porque luego para crear un pago en mercadopago mediante su [Payments API](https://www.mercadopago.com.ar/developers/es/reference/payments/_payments/post) necesitas especificar el mail. Si el dato del mail ya lo tienes (obtenido en un paso anterior al flujo), podrías especificar al brick ese dato y el mismo no va a solicitarlo en el formulario.

Saludos

**Reply by frithmatias** [reply]
Efectivamente Horacio con otro usuario test funcionó, disculpas por la confusión. Voy a pasarle este dato al brick como me recomiendas, ya que efectivamente lo tengo tengo en pasos anteriores al flujo. Muchas gracias por tu pronta respuesta!
